cli/bpmetadata/types_ui.go

```diff
@@ -0,0 +1,124 @@
+package bpmetadata
+
+// BlueprintUI is the top-level structure for holding UI specific metadata
+type BlueprintUI struct {
+	// The top-level input section that defines the list of properties and
+	// their sections on the deployment page
+	Input BlueprintUIInput `yaml:"input"`
+
+	// The top-level section for listing runtime (or solution output) information
+	// i.e. the console URL for the VM or a button to ssh into the VM etc based on
+	Runtime BlueprintUIOutput `yaml:"runtime"`
+
+	// The solution version corresponding to the version in the
+	// standard metadata file for keeping data in sync.
+	Version string `yaml:"version"`
```

Comment:
I assume this is just the semver string of the solution? How do we anticipate this going out of sync with standard metadata?

Reply:
It is the semver of the solution. It's unlikely but could happen in an error scenario where the standard file is updated with the latest variables but the display file is not.